symfony/security-bundle/5.3/config/packages/security.yaml

```diff
@@ -25,3 +25,12 @@ security:
     access_control:
         # - { path: ^/admin, roles: ROLE_ADMIN }
         # - { path: ^/profile, roles: ROLE_USER }
+
+when@test:
```

Comment:
Maybe add a small comment explaining that this is for performance purpose?

Reply:
I copied the note from the old documentation.